<launch>
  <arg name="participent" />
  <node name="record" pkg="rosbag" type="record" args="-a -O ~/experimental_results/participent_$(arg participent)/sloppy_joystick.bag"/>
  <include file="$(find behaviour_circuits)/launch/circuit.launch">
  <arg name="circuit" value="sloppy_joystick"/>
  </include> 
</launch>
